documentation/general/dash-sonic-hld.md

```diff
@@ -952,7 +974,7 @@ For the example configuration above, the following is a brief explanation of loo
 		c. Next lookup is in the mapping table and mapping table action here is "privatelink"
 		d. First Action for "privatelink" is 4to6 transposition
 		e. Packet gets transformed as: 
-		 	For Overlay SIP, using ENI's "pl_sip_encoding": "field:11:1:0x1:field:48:48:0x0a0b0c0d0a0b" -> Overlay SIP fd30:108:0:0a0b:0c0d0:0a0b:a01:102;	
+		 	For Overlay SIP, using ENI's "pl_sip_encoding": "0x0020000000000a0b0c0d0a0b/0x002000000000ffffffffffff" -> Overlay SIP fd30:108:0:0a0b:0c0d:0a0b:a01:102;	
 			Overlay DIP 2603:10e1:100:2::3402:206 (No transformation, provided as part of mapping)
 		f. Second Action is Static NVGRE encap with GRE key '100'. 
 		g. Underlay DIP shall be 50.2.2.6 (from mapping), Underlay SIP shall be 55.1.2.3 (from ENI)
```

Comment:
All of this explains processing in the outbound direction.  As discussed in the community meeting, please add the inbound processing details for both the ST and PL.  Thanks!

Reply:
Yes, it is updated in `section 2.3`, tagged as `ST/PL Inbound flow`

Reply:
@prsunny Sorry that this inbound processing explanation doesn't answer my question.  I was looking more from the point of view how you describe the VNET-to-VNET.  Specifically, in order to determine the packet-direction, we look at the "VNI" and then to the inner MAC to find the ENI to which the packet belongs to.  For ST/PL, in your example, if we are using NVGRE then the "key" is perhaps used to determine whether the packet is from Host or Network? Correct?  If that is the case, the VNET definition should be modified to introduce the concept of NVGRE "key".  Currently it only talks about VNI.  Should we make those clarifications?  Thanks!

Reply:
Hi @prsunny - couple of questions in bmv2 meeting today re: above.  
1 from @mhanif below:

Introduce concept / clarification of NVGRE key into documentation?

Reply:
@mhanif , it is already captured in section 2.1

"The pipeline shall parse the VNI, and for VM traffic, the VNI shall be a special reserved VNI. **Everything else shall be treated as as network traffic(RX)**."